tests/tracing/test_dnatracing_single_grain.py

```diff
@@ -145,6 +145,40 @@ def test_get_ordered_traces(dnatrace: dnaTrace, length: int, start: np.array, en
     )
 
 
+@pytest.mark.parametrize(
+    ("coordinate_list", "pixel_to_nm_scaling", "target_list"),
+    [
+        (
```

Comment:
I discovered a neat way of labelling tests so we can easily see what fails (wrote about it at [Pytest Parametrisation](https://ns-rse.github.io/posts/pytest-param/)).

Perhaps pedantic and would likely be useful going back through a lot of the code base which will be tedious but if we get in the habit of using it going forward I think it would be useful.

```suggestion
        pytest.param(
```